### ISSUE TITLE
about loss

### ISSUE THREAD

**Issue by FOURIERWANG** [jxgu1016/MNIST_center_loss.pytorch#17]
In my project experiment, the center loss does not decrease with the number of iterations.It seemingly irregular changes. I don't quite understand what's going on. I'd like to ask you about it.Thanks.

**Comment by jxgu1016** [comment]
If you want to ask questions, you should describe them detailedly.

**Comment by FOURIERWANG** [comment]
In my training, loss as a whole is declining, while softmax loss is declining, but the term  center loss is not declining,  there is no regular change about 10 to 20. I want to know how central loss should change.What details do you need to know?Thanks.
